_includes/xmlreference.md

```diff
@@ -158,7 +158,7 @@ Surface mesh consisting of vertices and (optional) of edges and triangles (only
 | Attribute | Type | Description | Default | Options |
 | --- | --- | --- | --- | --- |
 | name | string | Unique name for the mesh. | _none_ | none |
-| flip-normals | boolean | Deprectated. | `0` | none |
+| flip-normals | boolean | Deprecated. | `0` | none |
```

Comment:
That's a typo in the main repo, as the xmlreference is autogenerated by preCICE.

Reply:
Ah, I missed that. Here, it does not matter, as it will get updated anyway by the next submodule update. Feel free to update in preCICE.